src/Stratis.FederatedPeg.Features.FederationGateway/TargetChain/WithdrawalExtractor.cs

```diff
@@ -54,7 +54,7 @@ public IReadOnlyList<IWithdrawal> ExtractWithdrawalsFromBlock(Block block, int b
             return withdrawalsFromBlock;
         }
 
```

Comment:
cf my comment above, I don't understand that change

Reply:
see above reply.